src/Sarif.FunctionalTests/SarifConverterTestData/FxCop/FxCopReportResource.xml.sarif

```diff
@@ -5,7 +5,16 @@
       "toolInfo": {
         "name": "FxCop"
       },
-      "runInfo": {},
+      "runInfo": {
+        "fileInfo": {
+          "$(ProjectDir)/SpecifyCultureInfo/Microsoft.TeamFoundation.Build.Client.dll": [
```

Comment:
Note FxCop's special magic for creating deterministic file paths. At analysis time, $(ProjectDir) is expanded to create an absolute path for the analysis target that's relative to the project file which is driving analysis. This construct isn't literally a URL, so what do we do about it?